docs/book/src/topics/machinepools.md

```diff
@@ -104,4 +104,47 @@ spec:
       name: '{{ ds.meta_data.local_hostname }}'
       kubeletExtraArgs:
         cloud-provider: aws
-```
\ No newline at end of file
+```
+
+## Autoscaling
+
+[`cluster-autoscaler`](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) can be used to scale MachinePools up and down.
+Two providers are possible to use with CAPZ MachinePools: `clusterapi`, or `aws`.
```

Comment:
```suggestion
Two providers are possible to use with CAPA MachinePools: `clusterapi`, or `aws`.
```

Reply:
Good catch 😄

Reply:
oh 😄 sorry about that one! good catch indeed.